src/utils/__tests__/uploads.test.js

```diff
@@ -277,7 +277,7 @@ describe('util/uploads', () => {
         Date.now = jest.fn(() => now);
 
         test('should return item id correctly when item does not contain API options', () => {
-            expect(getDataTransferItemId(mockItem, rootFolderId)).toBe(`hi_0_${now}`);
+            expect(getDataTransferItemId(mockItem, rootFolderId)).toBe(`hi`);
```

Comment:
Since this is no longer an interpolated string, you can use single quotes instead of back ticks